@AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/core/fn_coreUpdate.sqf

```diff
@@ -83,12 +83,11 @@ if (_flightModel != "SFMPlus") then {
 };
 
 //Fuel
-private _apuFF  = 0;
```

Comment:
this needs to stay or else the value will never be reset to zero when the APU is off.